serverless.yml

```diff
@@ -33,7 +33,7 @@ functions:
       exclude:
         - ./**
     layers:
-      - arn:aws:lambda:eu-west-2:${self:custom.accountId}:layer:dev-es-commonfunctions:14
+      - arn:aws:lambda:eu-west-2:${self:custom.accountId}:layer:dev-es-common-functions:1
```

Comment:
Why this been changed back to 1?

Reply:
Ah. So from further inspection appears to be because the name has been changed from 'funk'. Can someone confirm this, please?

Reply:
When David renamed the function it reset the counter back to 1

Reply:
As I added a - to the name for standards the new one which gets deployed will start at 1 again.